exercises/object-oriented-programming/src/main/java/com/cbfacademy/cars/Cars.java

```diff
@@ -0,0 +1,70 @@
+package com.cbfacademy.cars;
+
+public class Cars {
+    // public String make = "Volvo";
+    // public String model = "V40";
+    // public int year = 2012;
+     String make = "Volvo";
+     String model = "V40";
+     int year = 2012;
+
+    // public Cars(String make, String model, int year){
+    //     this.make = make;
+    //     this.model = model;
+    //     this.year = year;
+    // }
+
+    public String toString(){
```

Comment:
As the `toString()` method belongs to the base `Object` class, you should mark this with the `@Override` annotation.